# Translation of Odoo Server.
# This file contains the translation of the following modules:
# 	* account_yodlee
# 
# Translators:
# Boris Stefanov <borkata@gmail.com>, 2020
# Rosen Vladimirov <vladimirov.rosen@gmail.com>, 2020
# aleksandar ivanov, 2020
# Albena Mincheva <albena_vicheva@abv.bg>, 2020
# Maria Boyadjieva <marabo2000@gmail.com>, 2020
# 
msgid ""
msgstr ""
"Project-Id-Version: Odoo Server saas~12.5+e\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2019-08-26 08:04+0000\n"
"PO-Revision-Date: 2019-08-26 09:34+0000\n"
"Last-Translator: Maria Boyadjieva <marabo2000@gmail.com>, 2020\n"
"Language-Team: Bulgarian (https://www.transifex.com/odoo/teams/41243/bg/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: \n"
"Language: bg\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "(Diagnostic: %r for URL %s)"
msgstr "(Диагностика: %r за URL %s)"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "Account"
msgstr "Сметка"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Account Already Registered (436):Your Auto-Registration could not be "
"completed because the site reports that your account is already registered."
"  Please log in to the site to confirm and then complete the site addition "
"process with the correct login information."
msgstr ""
"Профилът е вече регистриран (436):Вашата авторегистрация не можеше да се "
"извърши, тъй като сайтът докладва, че профилът Ви е вече регистриран. Моля, "
"влезте в сайта, за да потвърдите, след което завършете процеса по добавянето"
" в сайта с правилните данни за регистриране в системата."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Account Locked (407): We could not update your account because it appears "
"your account has been locked. This usually results from too many "
"unsuccessful login attempts in a short period of time. Please visit the site"
" or contact its customer support to resolve this issue.  Once done, please "
"update your account credentials in case they are changed."
msgstr ""
"Заключен профил (407): Не успяхме да актуализираме профила Ви, тъй като "
"очевидно профилът Ви е заключен. Това вероятно е следствие от многократни "
"неуспешни опити за влизане в рамките на кратък период от време. За да "
"разрешите проблема, моля посетете сайта или се свържете с отдел \"Поддръжка "
"клиенти\", след което актуализирайте акредитивите си, в случай, че са "
"претърпели промяна."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Account Not Found(408): We're sorry, we couldn't find any accounts for you "
"at the site. Please log in at the site and confirm that your account is set "
"up, then try again."
msgstr ""
"Профилът не е открит(408): Съжаляваме, не успяхме да открием никакви Ваши "
"профили в сайта. Моля, регистрирайте се в сайта и потвърдете, че профилът Ви"
" е вярно натроен, след което опитайте отново."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Account Reported Closed (422): We were unable to update your account "
"information because it appears one or more of your related accounts have "
"been closed.  Please deactivate or delete the relevant account and try "
"again."
msgstr ""
"Профилът е докладван като затворен (422): Не успяхме да актуализираме "
"информацията в профила Ви, тъй като явно един или повече от свързаните Ви "
"профили са били затворени. Моля, деактивирайте или изтрийте съответния "
"профил и опитайте отново."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Account Type Not Supported(417):The type of account we found is not "
"currently supported.  Please remove this site and add as a  manual account."
msgstr ""
"Този вид профил не се поддържа(417):Типът профил, който открихме, "
"понастоящем не се поддържа. Моля, отстранете този сайт и го добавете като "
"ръчен профил."

#. module: account_yodlee
#: model:ir.model.fields,help:account_yodlee.field_account_online_journal__yodlee_account_status
msgid "Active/Inactive on Yodlee system"
msgstr "Активен/неактивен в системата Yodlee"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "An Error has occurred (code %s)"
msgstr "Възникнала е грешка (код %s)"

#. module: account_yodlee
#: model:ir.model,name:account_yodlee.model_res_company
msgid "Companies"
msgstr "Компании"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "Could not make sense of parameters: %s"
msgstr ""

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Credential Re-Verification Required (402): We could not update your account "
"because your username and/or password were reported to be incorrect.  Please"
" re-verify your username and password."
msgstr ""
"Необходимо е повторно потвърждаване на акредитивите (402): Не можахме да "
"актуализираме профила Ви, тъй като е докладвано, ча потребителското Ви име "
"и/или парола не са правилни.  Моля потвърдете отново потребителското си име "
"и парола."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Credential Re-Verification Required (420):The site has merged with another. "
"Please re-verify your credentials at the site and update the same."
msgstr ""
"Необходимо е повторно потвърждаване на акредитивите (420):Сайтът се е слял с"
" друг такъв. Моля потвърдете повторно акредитивите си в сайта и го "
"актуализирайте."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "Error %s, message: %s"
msgstr ""

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "Error %s, message: %s, reference code: %s"
msgstr "Грешка %s, съобщение: %s, референтен код: %s"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Home Value Not Found (511): The site is unable to provide home value for "
"your property. We suggest you to delete this site."
msgstr ""
"Началната стойност не е открита (511): Сайтът не е наличен, за да предостави"
" началната стойност на Вашата собственост. Предлагаме Ви да изтриете този "
"сайт."

#. module: account_yodlee
#: model:ir.model,name:account_yodlee.model_account_online_journal
msgid "Interface for Online Account Journal"
msgstr ""

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Invalid Language Setting (421): The language setting for your site account "
"is not English. Please visit the site and change the language setting to "
"English."
msgstr ""
"Невалидна езикова настройка (421): Езиковата настройка на профила на сайта "
"Ви не е на английски. Моля посетете сайта и променете езиковата настройка на"
" английски."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Additional Authentication Enrollment Required (521) : Please "
"enroll in the new security authentication system, <Account Name> has "
"introduced. Ensure your account settings in <Cobrand> are updated with this "
"information."
msgstr ""
"MFA грешка: Необходимо е допълнително записване на достоверна информация "
"(521) : Моля впишете новата система за информационна сигурност, <Account "
"Name> е въведена. Уверете се, че настройките на профила Ви <Cobrand> са "
"актуализирани според тези данни."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Authentication Information Expired (524):We're sorry, the "
"authentication information you provided has expired. Please try again."
msgstr ""
"MFA грешка: Изтекла достоверна информация (524):Съжаляваме, предоставената "
"от Вас достоверна информация е изтекла. Моля, опитайте отново."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Authentication Information Incorrect (520):We're sorry, the site "
"indicates that the additional authentication information you provided is "
"incorrect. Please try updating your account again."
msgstr ""
"MFA грешка: Достоверната информация не е вярна (520):Съжаляваме, но сайтът "
"посочва, че допълнителната достоверна информация, която сте предоставили, не"
" е вярна. Моля опитайте се да актуализирате профила си отново."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Authentication Information Incorrect (523):We're sorry, the "
"authentication information you  provided is incorrect. Please try again."
msgstr ""
"MFA грешка: Достоверната информация не е вярна (523):Съжаляваме, "
"автентичната информация, която сте предоставили, не е вярна. Моля опитайте "
"отново."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Authentication Information Required (519): Your account was not "
"updated as your authentication information like security question and answer"
" was unavailable or incomplete. Please update your account settings."
msgstr ""
"MFA грешка: Необходима е достоверна информация (519): Профилът Ви не беше "
"актуализиран, тъй като изискуемата достоверна информация като въпросът и "
"отговорът, касаещи сигурността се оказаха липсващи или непълни. Моля "
"актуализирайте настройките на профила си."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Authentication Information Unavailable (518):Your account was not"
" updated as the required additional authentication information was "
"unavailable. Please try now."
msgstr ""
"MFA грешка: Достоверната информация не е налична (518):Профилът Ви не беше "
"актуализиран, тъй като се оказа, че изискуемата допълнителна информация за "
"удостоверяване не е налице. Моля опитайте сега."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Credential Re-Verification Required (526): We could not update "
"your account because your username/password or additional security "
"credentials are incorrect. Please try again."
msgstr ""
"MFA грешка: Необходимо е повторно потвърждаване на акредитивите (526): Не "
"успяхме да актуализираме профила Ви, тъй като потребителското Ви име/парола "
"или допълнителните Ви акредитиви за сигурност не са верни. Моля опитайте "
"отново."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Problem Updating Account (525): We could not update your account "
"for technical reasons. This type of error is usually resolved in a few days."
" We apologize for the inconvenience. Please try again later."
msgstr ""
"MFA грешка: Проблем с актуализацията на профила (525): Не успяхме да "
"актуализираме профила Ви по технически причини. Този тип грешка обикновено "
"се отстранява в рамките на няколко дни. Извиняваме са за причиненото "
"неудобство. Моля опитайте отново по-късно."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Request Timed Out (522) :Your request has timed out as the "
"required security information was unavailable or was not provided within the"
" expected time. Please try again."
msgstr ""
"MFA грешка: Запитването е изтекло (522): Запитването Ви е изтекло, тъй като "
"изискуемината поверителна информация не е налична или не е предоставена в "
"рамките на очакваното време. Моля опитайте отново."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Site Device Information Expired(509): We're sorry, we can't "
"update your account because your token is no longer valid at the site. "
"Please update your information and try again, or contact customer support."
msgstr ""
"MFA грешка: Информацията за устройството на сайта е изтекла(509): "
"Съжаляваме, не можем да актуализираме профила Ви, тъй като токенът Ви вече "
"не е валиден на сайта. Моля актуализирайте информацията си и опитайте отново"
" или се свържете с отдел \"Поддръжка клиенти\"."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Multiple User Logins(416): We attempted to update your account, but another "
"session was already established at the same time.  If you are currently "
"logged on to this account directly, please log off and try after some time"
msgstr ""
"Множество потребителски регистрации (416): Опитахме се да актуализираме "
"профила Ви, но междувременно беше създадена друга сесия. Ако в момента сте "
"влезли в този профил директно, моля излезте и опитайте отново след известно "
"време."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"New Login Information Required(506):We're sorry, to log in to this site, you"
" need to provide additional information. Please update your account and try "
"again."
msgstr ""
"Необходима е нова информация за регистрация (506): Съжаляваме, но за да "
"влезете в този сайт, е необходимо да предоставите допълнителна информация. "
"Моля актуализирайте профила си и опитайте пак."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"No Account Found (423): We were unable to detect an account. Please verify "
"that you account information is available at this time and If the problem "
"persists, please contact customer support at online@odoo.com for further "
"assistance."
msgstr ""
"Не е открит профил (423): Не успяхме да засечем профил. Моля потвърдете, че "
"информацията в профила Ви е налична към момента и, ако проблемът не се "
"отстранява, за да получите допълнителна помощ, моля свържете се с отдел "
"\"Поддръжка клиенти\" на online@odoo.com."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"No Payees Found(512):Your request cannot be completed as no payees were "
"found in your account."
msgstr ""
"Не са открити предявители(512):Запитването Ви не може да се завърши, защото "
"в профила Ви не бяха открити предявители."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "Online institution"
msgstr ""

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "Please configure an email in the company settings."
msgstr "Моля конфигурирайте имейл в настройките на компанията."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account (406): We could not update your account because the"
" site requires you to perform some additional action. Please visit the site "
"or contact its customer support to resolve this issue. Once done, please "
"update your account credentials in case they are changed else try again."
msgstr ""
"Проблем с актуализацията на профила (406): Не можахме да актуализираме "
"профила Ви, защото сайтът изисква от Вас да извършите няколко допълнителни "
"стъпки. Моля, за да разрешите проблема, посетете сайта или се свържете с "
"отдел \"Поддръжка клиенти\", след което актуализирайте акредитивните си "
"писма и, в случай, че са променени, опитайте отново."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account (517): We are sorry, there was a technical problem "
"updating your account. Please try again."
msgstr ""

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(401):We're sorry, your request timed out. Please "
"try again."
msgstr ""
"Проблем с актуализацията на профила (401):Съжаляваме, времето на заявката Ви"
" изтече. Моля опитайте отново."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(403):We're sorry, there was a technical problem "
"updating your account. This kind of error is usually resolved in a few days."
" Please try again later."
msgstr ""
"Проблем с актуализацията на профила(403):Съжаляваме, при актуализацията на "
"профила Ви възникна технически проблем. Този тип грешка обикновено се "
"отстранява в рамките на няколко дни. Моля опитайте по-късно."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(404):We're sorry, there was a technical problem "
"updating your account. Please try again later."
msgstr ""
"Проблем с актуализацията на профила(404):Съжаляваме, при актуализацията на "
"профила Ви възникна технически проблем. Моля опитайте отново по-късно."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(409): We could not update your account because the "
"end site is experiencing technical difficulties."
msgstr ""
"Проблем с актуализацията на профила(409): Не можахме да актуализираме "
"профила Ви, тъй като крайният сайт изпитва технически затруднения."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(412): We could not update your account because the "
"site is experiencing technical difficulties."
msgstr ""
"Проблем с актуализацията на профила(412): Не можахме да актуализираме "
"профила Ви, защото сайтът изпитва технически затруднения."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(413):We're sorry, we couldn't update your account "
"at the site because of a technical issue. This type of problem is usually "
"resolved in a few days. Please try again later."
msgstr ""
"Проблем с актуализацията на профила(413):Съжаляваме, не можахме да "
"актуализираме профила Ви, заради възникнал технически проблем. Този тип "
"проблем обикновено се отстранява в рамките на няколко дни. Моля опитайте "
"отново по-късно."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(415): We could not update your account because the "
"site is experiencing technical difficulties."
msgstr ""
"Проблем с актуализацията на профила(415): Не можахме да актуализираме "
"профила Ви, защото сайтът изпитва технически затруднения."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(418): We could not update your account because the "
"site is experiencing technical difficulties. Please try later."
msgstr ""
"Проблем с актуализацията на профила(418): Не можахме да актуализираме "
"профила Ви, защото сайтът изпитва технически затруднения. Моля опитайте по-"
"късно."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(419):We're sorry, we couldn't update your account "
"because of unexpected variations at the site. This kind of problem is "
"usually resolved in a few days. Please try again later."
msgstr ""
"Проблем с актуализацията на профила(419):Съжаляваме, не можахме да "
"актуализираме Вашия профил, заради неочаквани варианти в сайта. Този тип "
"проблем обикновено се отстранява в рамките на няколко дни. Моля опитайте "
"отново по-късно."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(425): We could not update your account because the "
"site is experiencing technical difficulties. Please try later."
msgstr ""
"Проблем с актуализацията на профила(425): Не можахме да актуализираме "
"профила Ви, тъй като сайтът изпитва технически затруднения. Моля опитайте "
"по-късно."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(426): We could not update your account for "
"technical reasons. This type of error is usually resolved in a few days. We "
"apologize for the inconvenience."
msgstr ""
"Проблем с актуализацията на профила(426): Не можахме да актуализираме "
"профила Ви по технически причини. Този тип грешка обикновено се разрешава в "
"рамките на няколко дни. Извиняваме се за причиненото неудобство."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(507):We're sorry, Yodlee has just started providing"
" data updates for this site, and it may take a few days to be successful as "
"we get started. Please try again later."
msgstr ""
"Проблем с актуализацията на профила(507):Съжаляваме, Yodlee тъкмо започна да"
" предоставя актуализация на данни за този сайт и е възможно да отнеме "
"няколко дни до успешното приключване. Моля опитайте отново по-късно."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Property Record Not Found (510): The site is unable to find any property "
"information for your address. Please verify if the property address you have"
" provided is correct."
msgstr ""
"Не е открит запис на собствеността (510): Сайтът е неспособен да открие "
"каквато и да било информация за собствеността на адресите Ви. Моля "
"потвърдете, ако адресът на собствеността, който сте предоставили е верен."

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_account_online_provider__provider_type
msgid "Provider Type"
msgstr "Тип доставчик"

#. module: account_yodlee
#: model:ir.model,name:account_yodlee.model_account_online_provider
msgid "Provider for online account synchronization"
msgstr "Доставчик синхронизиране на онлайн профили"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Re-Verification Required (429): We could not update your account due to the "
"site requiring you to verify your personal information. Please log in to the"
" site and update the fields required."
msgstr ""
"Изисква се повторно потвърждение (429): Не можахме да актуализираме профила "
"Ви, тъй като сайтът изисква от Вас да потвърдите личната си информация. "
"Моля, влезте в сайта и актуализирайте полетата, които се изискват."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Re-verification Required (427): We could not update your account due to the "
"site requiring you to view a new promotion. Please log in to the site and "
"click through to your account overview page to update the account.  We "
"apologize for the inconvenience."
msgstr ""
"Изисква се повторно потвърждение (427): Не можахме да актуализираме профила "
"Ви, тъй като сайтът изисква от Вас да видите нова промоция. Моля, влезте в "
"сайта и натиснете бутоните в страницата с изгледа на профила си, за да го "
"актуализирате. Извиняваме се за причиненото неудобство."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Re-verification Required (428): We could not update your account due to the "
"site requiring you to accept a new Terms & Conditions. Please log in to the "
"site and read and accept the T&C."
msgstr ""
"Изисква се повторно потвърждение (428): Не можем да актуализираме профила "
"Ви, тъй като сайтът изисква от Вас да приемете новите срокове&условия. Моля,"
" влезте в сайта, за да ги прочетете и одобрите С&У."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Registration Requires Attention (433): Auto registration is not complete. "
"Please complete your registration at the end site. Once completed, please "
"complete adding this account."
msgstr ""
"Регистрацията изисква внимание (433): Авторегистрацията не е завършена. "
"Моля, завършете регистрацията си докрай. След като приключите, моля, "
"завършете добавянето на този профил."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Registration Requires Attention (434): Your Auto-Registration could not be "
"completed and requires further input from you.  Please re-verify your "
"registration information to complete the process."
msgstr ""
"Регистрацията изисква внимание (434): Вашата авторегистрация не можа да се "
"извърши и изисква последващо въвеждане. Моля потвърдете отново свързаната с "
"регистрацията си информация, за да завършите процеса."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Registration Requires Attention (435): Your Auto-Registration could not be "
"completed and requires further input from you.  Please re-verify your "
"registration information to complete the process."
msgstr ""
"Регистрацията изисква внимание (435): Вашата авторегистрация не можа да се "
"извърши и изисква последващо въвеждане. Моля потвърдете отново свързаните с "
"регистрацията си данни, за да завършите процеса."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Request Timed Out (508): We are sorry, your request timed out due to "
"technical reasons. Please try again."
msgstr ""
"Запитването прекъснато (508): Съжаляваме, запитването Ви е прекъснато поради"
" технически причини. Моля опитайте отново."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Requested Account Type Not Found (414): We could not find your requested "
"account. You may have selected a similar site under a different category by "
"accident in which case you should select the correct site."
msgstr ""
"Поисканият вид акаунт не е открит (414): Не можахме да открием поискания от "
"Вас акаунт. Възможно е по случайност да сте избрали подобен сайт в друга "
"категория. За целта трябва да изберете правилния сайт."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "Server not reachable, please try again later"
msgstr ""

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Site Down for Maintenance(424):We were unable to update your account as the "
"site is temporarily down for maintenance. We apologize for the "
"inconvenience.  This problem is typically resolved in a few hours. Please "
"try later."
msgstr ""
"Сайтът е свален за поддръжка(424):Не сме в състояние да актуализираме "
"акаунта Ви, тъй като сайтът е временно свален за поддръжка. Извиняваме се за"
" причиненото неудобство. Този проблем обикновено се решава в рамките на "
"няколко часа. Моля опитайте по-късно."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Site No Longer Available (411):The site no longer provides online services "
"to its customers.  Please delete this account."
msgstr ""
"Сайтът вече не е наличен (411):Сайтът вече не предоставя онлайн услуги на "
"своите слиенти. Моля изтрийте този акаунт."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Site No Longer Supported (430):This site is no longer supported for data "
"updates. Please deactivate or delete your account. We apologize for the "
"inconvenience."
msgstr ""
"Сайтът вече не се поддържа (430):Този сайт вече не се поддържа за "
"актуализация на данни. Моля деактивирайте или изтрийте акаунта си. "
"Извиняваме се за причиненото неудобство."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Site Not Supported (505): We currently does not support the security system "
"used by this site. We apologize for any inconvenience. Check back "
"periodically if this situation has changed."
msgstr ""
"Сайтът не се поддържа (505): Понастоящем не поддържаме системата за "
"сигурност, използвана от този сайт. Извиняваме се за причиненото неудобство."
" Проврявайте периодично дали тази ситуация се е променила."

#. module: account_yodlee
#: model:ir.actions.client,name:account_yodlee.yodlee_widget
msgid "Synchronization process completed"
msgstr ""

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#: code:addons/account_yodlee/models/yodlee.py:0
#: code:addons/account_yodlee/models/yodlee.py:0
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "Timeout: the server did not reply within 30s"
msgstr "Таймаут: сървърът не реагира в рамките на 30 сек."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Update Request Canceled(405):Your account was not updated because you "
"canceled the request."
msgstr ""
"Заявката за актуализация отменена(405):Вашата сметка не беше актуализирана, "
"защото отменихте заявката."

#. module: account_yodlee
#: model:ir.model.fields.selection,name:account_yodlee.selection__account_online_provider__provider_type__yodlee
msgid "Yodlee"
msgstr "Yodlee"

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_account_online_journal__yodlee_account_status
msgid "Yodlee Account Status"
msgstr ""

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_account_online_journal__yodlee_status_code
msgid "Yodlee Status Code"
msgstr ""

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_res_company__yodlee_user_access_token
msgid "Yodlee access token"
msgstr "Токен за достъп на Yodlee"

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_res_company__yodlee_user_login
msgid "Yodlee login"
msgstr "Влизане в Yodlee"

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_res_company__yodlee_user_password
msgid "Yodlee password"
msgstr "Yodlee парола"

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_res_company__yodlee_access_token
msgid "access_token"
msgstr "access_token"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "message: %s"
msgstr ""
